utility/build-homebrew.sh

```diff
@@ -23,7 +23,7 @@ pip install cumulusci homebrew-pypi-poet || exit 1
 echo " "
 echo "=> Collecting dependencies and generating resource stanzas..."
 echo " "
-poet cumulusci > "$RES_FILE"
+poet cumulusci | awk '/resource "cumulusci"/{c=5} !(c&&c--)' > "$RES_FILE"
```

Comment:
Can you add a comment to remind me what this does without having to refer to awk docs?

Reply:
Done.